site/docs-md/cordova/index.md

```diff
@@ -0,0 +1,46 @@
+# Cordova and PhoneGap
+
+Apache Cordova (and Adobe PhoneGap), created in 2008, is an open source framework that enables web developers to use their HTML, CSS, and JavaScript content to create a native application for a variety of mobile and desktop platforms.
+
+For more details on the history of Cordova and more details on how it works, [please see here](https://ionicframework.com/enterprise/resources/articles/what-is-apache-cordova).
+
+## Why Create a New Project?
+
+The open source space is filled with new projects that build on top of older projects, making tangible improvements that can't be done without radically changing the original product. This is what Capacitor would have required. The Ionic team felt like this wasn't possible with Cordova for technical and political reasons. Whether that is right or wrong, that is the conclusion the team came to. That said, Ionic still uses Cordova heavily and will continue to invest in the platform for a long time to come.
+
+On the plus side, Ionic now controls almost all of its stack. When you build an Ionic app and use Capacitor, we control the native runtime layer, the UI controls, and the “framework” used to build the controls ([Stencil](https://stenciljs.com/)). The only part we don’t control is the frontend framework you use on top (Angular, React, Vue, or nothing). This is significant: If there’s an issue in any part of the stack that we control, we can fix it right away. Capacitor has already proven to be a worthwhile investment - it’s enabling us to build a stronger Ionic and focus on what we do uniquely well.
+
+## Differences between Capacitor and Cordova
+
+Capacitor has a few key differences that require web developers, previously used to Cordova's approach, to change app development workflows.
+
+### Native Project Management
+
+Capacitor considers each platform project a _source asset_ instead of a _build time asset_. That means you'll check your Xcode and Android Studio projects into source control, as well as use those IDEs when necessary for platform-specific configuration and running/testing.
+
+This change in approach has a few implications. First, Capacitor does not use `config.xml` or a similar custom configuration for platform settings. Instead, configuration changes are made by editing the appropriate platform-specific configuration files directly, such as `AndroidManifest.xml` for Android and `Info.plist` for Xcode.
```

Comment:
Well, some settings are in `capacitor.config.json`

Reply:
this was originally written by max long ago - thanks for the update! noted.